### ISSUE TITLE
Deprecation `types` option

### ISSUE THREAD

**Issue by Frexuz** [ubilabs/geocomplete#298]
https://developers.google.com/maps/documentation/javascript/places?csw=1#deprecation

Should change your API accordingly :)

**Comment by chrisli30** [comment]
I really like this library which saved me a lot of work on UI so I researched Google and made "types" work again in #301 .

1. Google has changed the parameter from "types" to "type", but most of type values remain unchanged.

2. Regardless, the only change we need to make is to explicitly call a new method setTypes() of AutoComplete. For example, I added this line right after autocomplete object initialization.
this.autocomplete.setTypes(types);

**Comment by chrisli30** [comment]
I did test locally but didn't check in test code due to lake of cleanness. Hope it helps!

**Comment by aemkei** [comment]
This is true for "text search requests" but looks like, AutoComplete is not affected: https://developers.google.com/maps/documentation/javascript/3.exp/reference#AutocompleteOptions
